README.md

```diff
@@ -1,7 +1,15 @@
-# **Cemu - Wii U emulator**
-
-[![Build Process](https://github.com/cemu-project/Cemu/actions/workflows/build.yml/badge.svg)](https://github.com/cemu-project/Cemu/actions/workflows/build.yml)
-[![Discord](https://img.shields.io/discord/286429969104764928?label=Cemu&logo=discord&logoColor=FFFFFF)](https://discord.gg/5psYsup)
+<div align="center">
+<h1><strong>Cemu - Wii U emulator</strong></h1>
+</div>
+
+<div align="center">
+<a href="https://github.com/cemu-project/Cemu/actions/workflows/build.yml">
+ <img src="https://github.com/cemu-project/Cemu/actions/workflows/build.yml/badge.svg" alt="Build Process"/>
+ </a>
+ <a href="https://discord.gg/5psYsup">
+  <img src="https://img.shields.io/discord/286429969104764928?label=Cemu&logo=discord&logoColor=FFFFFF" alt="Discord"/>
+ </a>
+</div>
```

Comment:
Using HTML tags in Markdown files is not supported by a lot of parsers. Also, why did you replace Markdown-style images with HTML `<img>` tags?